jupyterlab/labbox_ephys_widgets_jp/src/pluginWidgets.tsx

```diff
@@ -196,47 +196,64 @@ interface PluginComponentWrapperProps {
 }
 
 const PluginComponentWrapper: FunctionComponent<PluginComponentWrapperProps> = ({plugin, hither, sortingObject, recordingObject, sortingInfo, recordingInfo, plugins, calculationPool, model}) => {
-  let sorting: Sorting
-  let recording: Recording
-  if (sortingObject.sorting_format) {
-    sorting = {
+  const [sorting, setSorting] = useState<Sorting | null>(null)
```

Comment:
Reorganize how curation is synchronized between the jupyter model and the react state. The sorting is now a state variable and we reset it whenever the curation has changed.